### ISSUE TITLE
Option to enter multiple arguments at once (optimizer argments in particular)

### ISSUE THREAD

**Issue by MarvyD** [derrian-distro/LoRA_Easy_Training_Scripts#278]
In Kohya GUI when i switch optimizer, let's say to Prodigy  I take arguments I use from .txt file and just insert it in "optimizer arguments" filed, 
"decouple=True weight_decay=0.01 d_coef=0.8 betas=0.9,0.99 use_bias_correction=True" , but as far as I see there is no way to quickly add multiple arguments at once, and I have to first copy argument name and then argument value manually for each argument I need (In case of Prodigy 8 times).

It would be great If program could take arguments line like this "decouple=True weight_decay=0.01 d_coef=1.0 betas=0.9,0.99 use_bias_correction=True" and split it in different fields automatically. I don't know how hard it would be to implement, another option could be an option to have different presets for Optimizer Arguments you could save and switch to when needed. Or just a filed somewhere at the bottom which would take arguments in default "Kohya" format.

**Comment by Jelosus2** [comment]
Why dont you just save the toml with the settings and import it whenever you need it?

**Comment by MarvyD** [comment]
Quite often I need to switch only optimizer, without having to set up image folders and other setting again. I also checked if removing everything but optimizer related setting from toml file would make it act like present and change only these settings, but it resets to default all settings which are not included in toml file.

**Comment by Jelosus2** [comment]
Why would you do that? Just open the UI in a clean state, do your changes and save the toml?

**Comment by derrian-distro** [comment]
If truly necessary, I can probably add an option to the optimizer args for importing and exporting json files with the inputs independently of the main toml system. I can't give an exact time frame, but it wouldn't be too hard to implement

**Comment by MarvyD** [comment]
That would be great, I'm experimenting with different optimizers and how they affect training, so being able to switch them quickly without affecting other settings is an important option in this case. You could even save whole optimizer field including learning rate.

**Comment by derrian-distro** [comment]
I have added functionality to load a json file for any of the "optional args" fields, such as the optimizer, network, and extra args fields, so closing this
